### PR TITLE
[DO_NOT_MERGE] Microphone streaming with a 1 minute duration.

### PR DESCRIPTION
**Pull request by nirupa-kumar**
@nnegrey Please review

### CODE REVIEW

**Comment by nirupa-kumar** [comment]
fixing the styling issues.

File: speech/cloud-client/src/main/java/com/example/speech/Recognize.java [hunk 1]
```diff
@@ -704,6 +717,98 @@ public void onCompleted() {
...
+
+      clientStream.send(request);
+      // SampleRate:16000Hz, SampleSizeInBits: 16, Number of channels: 1, Signed: true,
+      // bigEndian: false
+      AudioFormat audioFormat = new AudioFormat(16000, 16, 1, true, false);
+      DataLine.Info targetInfo =
+          new Info(
+              TargetDataLine.class,
+              audioFormat); // Set the system information to read from the microphone audio stream
+      TargetDataLine targetDataLine;
```
Comment (nnegrey) [inline speech/cloud-client/src/main/java/com/example/speech/Recognize.java]: Delete this line (see below comment)

File: speech/cloud-client/src/main/java/com/example/speech/Recognize.java [hunk 2]
```diff
@@ -704,6 +717,98 @@ public void onCompleted() {
...
+      int durationMillSec = 60 * 1000; // 60 seconds
+      if (!AudioSystem.isLineSupported(targetInfo)) {
+        System.out.println("Microphone not supported");
+        System.exit(0);
+      }
+      //Target data line captures the audio stream the microphone produces.
+      targetDataLine = (TargetDataLine) AudioSystem.getLine(targetInfo);
```
Comment (nnegrey) [inline speech/cloud-client/src/main/java/com/example/speech/Recognize.java]: TargetDataLine targetDataLine = ...

File: speech/cloud-client/src/main/java/com/example/speech/Recognize.java [hunk 3]
```diff
@@ -704,6 +717,98 @@ public void onCompleted() {
...
+      clientStream.send(request);
+      // SampleRate:16000Hz, SampleSizeInBits: 16, Number of channels: 1, Signed: true,
+      // bigEndian: false
+      AudioFormat audioFormat = new AudioFormat(16000, 16, 1, true, false);
+      DataLine.Info targetInfo =
+          new Info(
+              TargetDataLine.class,
+              audioFormat); // Set the system information to read from the microphone audio stream
+      TargetDataLine targetDataLine;
+      int durationMillSec = 60 * 1000; // 60 seconds
```
Comment (nnegrey) [inline speech/cloud-client/src/main/java/com/example/speech/Recognize.java]: Delete this line

File: speech/cloud-client/src/main/java/com/example/speech/Recognize.java [hunk 4]
```diff
@@ -704,6 +717,98 @@ public void onCompleted() {
...
+      targetDataLine.start();
+      System.out.println("Start speaking");
+      long startTime = System.currentTimeMillis();
+      // Audio Input Stream
+      AudioInputStream audio = new AudioInputStream(targetDataLine);
+      while (true) {
+        long estimatedTime = System.currentTimeMillis() - startTime;
+        byte[] data = new byte[6400];
+        audio.read(data);
+        if (estimatedTime > durationMillSec) {
```
Comment (nnegrey) [inline speech/cloud-client/src/main/java/com/example/speech/Recognize.java]: if (estimatedTime > 60000) {  // 60 seconds

**Pull request merged**